.github/workflows/ci-pr.yml

```
@@ -0,0 +1,31 @@
+        uses: actions/checkout@v4
+
+      - name: Extract metadata (tags) for Docker
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: ${{ secrets.DOCKER_USERNAME }}/loanportal
+          tags: |
+            type=raw,value=pr-${{ github.event.number }}
+
+      - name: Build Docker image
+        uses: docker/build-push-action@v4
```

actionlint (1.7.10)

[error] the runner of "docker/build-push-action@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)